src/Gui.cc

```diff
@@ -57,6 +60,10 @@ Errors Gui::Load(ElementPtr _sdf)
   this->dataPtr->fullscreen = _sdf->Get<bool>("fullscreen",
       this->dataPtr->fullscreen).first;
 
+  Errors pluginErrors = loadUniqueRepeated<Plugin>(_sdf, "plugin",
```

Comment:
Should we use `loadRepeated` instead since plugin names are not necessarily unique (see https://github.com/ignitionrobotics/sdformat/pull/721 and https://github.com/ignitionrobotics/sdformat/issues/722)

Reply:
Done in Done in 980d5d2